standards.py

```diff
@@ -16,6 +16,7 @@
 parser.add_argument("--fix", action="store_true")
 parser.add_argument("--yamato", action="store_true")
 
+parser.add_argument("--verbosity", default="minimal")
```

Comment:
this is the real addition in the PR, other file changes are simply results from running `./standards.py --fix` locally.